tests/python/driver/tvmc/test_model.py

```diff
@@ -45,6 +46,31 @@ def test_tvmc_workflow(keras_simple):
     assert "output_0" in result.outputs.keys()
 
 
+def test_tvmc_workflow_use_vm(keras_simple):
+    pytest.importorskip("tensorflow")
+    import tensorflow as tf
+
+    # Reset so the input name remains consistent across unit test runs
+    tf.keras.backend.clear_session()
+
+    tvmc_model = tvmc.load(keras_simple)
+    tuning_records = tvmc.tune(tvmc_model, target="llvm", enable_autoscheduler=True, trials=2)
+    tvmc_package = tvmc.compile(
+        tvmc_model, tuning_records=tuning_records, target="llvm", use_vm=True
+    )
+
+    input_dict = {"input_1": np.random.uniform(size=(1, 32, 32, 3)).astype("float32")}
+    result = tvmc.run(tvmc_package, device="cpu", end_to_end=True, inputs=input_dict)
+
+    assert type(tvmc_model) is TVMCModel
+    assert type(tvmc_package) is TVMCPackage
+    assert type(result) is TVMCResult
+    assert path.exists(tuning_records)
+    assert type(result.outputs) is dict
+    assert type(result.times) is BenchmarkResult
+    assert "output_0" in result.outputs.keys()
+
+
```

Comment:
I think we should also modify `test_save_load_model` to make sure that saving and loading with `use_vm=True` works as expected.